sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/Components15Gvwi3SchemasTasksstatePropertiesTasksPropertiesEntityrecognitiontasksItemsAllof1.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.ai.textanalytics.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The
+ * Components15Gvwi3SchemasTasksstatePropertiesTasksPropertiesEntityrecognitiontasksItemsAllof1
+ * model.
+ */
+@Fluent
+public class Components15Gvwi3SchemasTasksstatePropertiesTasksPropertiesEntityrecognitiontasksItemsAllof1 {
```

Comment:
The name of this class is messed up?

Reply:
Yes. Autogenerated.
I think it can be removed. Want a confirm from @suhas92

Reply:
Those classes are duplicate classes. Somehow the autogene/swagger mess these up. I removed them already.

Reply:
Manually removed?
If yes, we should check with the autorest crew on this and not do manual updates to generated code.

Reply:
it is the swagger issue. I think it is safe to remove since they are useless.